docs/guides/05-introduction-to-styling.md

```diff
@@ -4,75 +4,55 @@ This guide walks through making a variety of visualizations for points, lines, a
 
 ### Supported Color Spaces
 
-* [HEX](https://carto.com/developers/carto-vl/reference/#cartoexpressionshex)
-* [RGB](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgb)
-* [RGBA](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgba)
-* [Named Colors](https://carto.com/developers/carto-vl/reference/#cartoexpressionsnamedcolor)
-* [HSV](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsv)
-* [HSVA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsva)
-* [HSL](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsl)
-* [HSLA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsla)
-* [CIELAB](https://carto.com/developers/carto-vl/reference/#cartoexpressionscielab)
+Supported color spaces are [HEX](https://carto.com/developers/carto-vl/reference/#cartoexpressionshex), [RGB](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgb), [RGBA](https://carto.com/developers/carto-vl/reference/#cartoexpressionsrgba), [Named Colors](https://carto.com/developers/carto-vl/reference/#cartoexpressionsnamedcolor), [HSV](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsv), [HSVA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsva), [HSL](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsl), [HSLA](https://carto.com/developers/carto-vl/reference/#cartoexpressionshsla), [CIELAB](https://carto.com/developers/carto-vl/reference/#cartoexpressionscielab).
 
 In addition to these [supported color spaces](http://cartovl-tabs.developers.carto-staging.com/developers/carto-vl/examples/#example-color-spaces), you can also use our [CARTOcolors](https://carto.com/carto-colors/) schemes directly.
 
-## Basic Styling Properties
+### Styling Properties
 
-* Color
-* Width
-* Stroke color
-* Stroke width
-* Resolution
-* Opacity
-
-### Color
+#### Color
 Use the `color` property to define the fill color of a point, line or polygon.
 
 ```
 color: #F24440
 ```
 
-### Width
+#### Width
 Use the `width` property to define a point size or line width.
 
 ```
 width: 5
 ```
 
-### Stroke Color
+#### Stroke Color
 Use the `strokeColor` property to define the color of a point or polygon stroke.
 
 ```
 strokeColor: #F24440
 ```
 
-### Stroke Width
+#### Stroke Width
 Use the `strokeWidth` property to define the size of a point or polygon stroke width.
 
 ```
 strokeWidth: 3
 ```
 
-### Resolution
+#### Resolution
 Use `resolution` to aggregate points into clusters.
 
 ```
 resolution: 5
 ```
 
-### Opacity
-Set a feature´s opacity with the alpha channel or inside of a `ramp` [expression](https://carto.com/developers/carto-vl/guides/introduction-to-expressions/).
+### Filter
+Use `filter` to show only the features that match the specified expression.
 
 ```
-color: rgba(232,66,244,0.5)
-```
-```
-color: opacity(ramp($animals,Prism),0.5)
+filter: between($numfloors, 10, 120)
 ```
 
-## Color by value
```

Comment:
This was done on purpose. The separation between the basic styling properties and the more "advanced" ones. We want to revert this change.

Reply:
The problem here is that we have lots of levels so things are not looking good in the dev-center. With this approach, I think it is clearer (at least for me). But feel free to update it.

Reply:
**Form follows function.** We shouldn't compromise the way we teach our product because the styling is not ideal. Let's not change the content, let's improve the styling.